components/script/dom/webidls/VR.webidl

```diff
@@ -0,0 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// https://w3c.github.io/webvr/#interface-navigator
+interface VR: EventTarget {
+  Promise<sequence<VRDisplay>> getDisplays();
+  //readonly attribute FrozenArray<VRDisplay> activeVRDisplays;
+  readonly attribute boolean vrEnabled;
```

Comment:
IMO getAvailability or isEnabled() is a useful API to check VR availability because it doesn't load VRDisplays. I mean that you can check availability using the getDisplays promise but in every browser that probably wakes up the lazy loaded WebVR thread, resources, and more. In a webpage you may want to only know availability without loading all those resources.